lms/static/sass/shared-v2/_components.scss

```diff
@@ -0,0 +1,49 @@
+// TODO: tabs should be added to the Pattern Library
+.tabs {
+    @include clearfix();
+    @extend %reset-lists;
```

Comment:
@clrux I've heard of this ordering but don't know what it is. Do you have a pointer to a description of the suggested order?